build.gradle

```diff
@@ -36,11 +36,11 @@ buildscript {
 }
 
 ext {
-    groupId = 'de.cyface'
```

Comment:
Steht jetzt direkt in der publish.gradle, da nur dort benutzt.

Ansonsten müssen wir auf die Variable mit rootProject.ext.var zugreifen, d.h. wenn man das Backend als submodule eingebunden hat kann man versehentlich die groupId in der app/build.gradle/ext (die denn das rootProject ist) überschreiben. Dann wird ggf. versehentlich automatisch ein Package mit dem falschen Namen (groupId) in die Registry gepusht, woraus wir es nicht mehr entfernen können.